tracee/consts.go

```diff
@@ -836,4 +854,22 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
 	VfsWritevEventID:           {{Type: "const char*", Name: "pathname"}, {Type: "dev_t", Name: "dev"}, {Type: "unsigned long", Name: "inode"}, {Type: "unsigned long", Name: "vlen"}, {Type: "off_t", Name: "pos"}},
 	MemProtAlertEventID:        {{Type: "alert_t", Name: "alert"}},
 	SchedProcessExitEventID:    {},
+	PidfdSendSignal:            {{Type: "int", Name: "pidfd"}, {Type: "int", Name: "sig"}, {Type: "siginfo_t*", Name: "info"}, {Type: "unsigned int", Name: "flags"}},
+	IoUringSetup:               {{Type: "unsigned int", Name: "entries"}, {Type: "struct io_uring_params*", Name: "p"}},
+	IoUringEnter:               {{Type: "unsigned int", Name: "fd"}, {Type: "unsigned int", Name: "to_submit"}, {Type: "unsigned int", Name: "min_complete"}, {Type: "unsigned int", Name: "flags"}, {Type: "sigset_t*", Name: "sig"}},
+	IoUringRegister:            {{Type: "unsigned int", Name: "fd"}, {Type: "unsigned int", Name: "opcode"}, {Type: "void*", Name: "arg"}, {Type: "unsigned int", Name: "nr_args"}},
+	OpenTree:                   {{Type: "int", Name: "dfd"}, {Type: "const char*", Name: "filename"}, {Type: "unsigned int", Name: "flags"}},
+	MoveMount:                  {{Type: "int", Name: "from_dfd"}, {Type: "const char*", Name: "from_path"}, {Type: "int", Name: "to_dfd"}, {Type: "const char*", Name: "to_path"}, {Type: "unsigned int", Name: "flags"}},
+	Fsopen:                     {{Type: "const char*", Name: "fsname"}, {Type: "unsigned int", Name: "flags"}},
+	Fsconfig:                   {{Type: "int", Name: "fs_fd"}, {Type: "unsigned int", Name: "cmd"}, {Type: "const char*", Name: "key"}, {Type: "const void*", Name: "value"}, {Type: "int", Name: "aux"}},
+	Fsmount:                    {{Type: "int", Name: "fsfd"}, {Type: "unsigned int", Name: "flags"}, {Type: "unsigned int", Name: "ms_flags"}},
+	Fspick:                     {{Type: "int", Name: "dirfd"}, {Type: "const char*", Name: "pathname"}, {Type: "unsigned int", Name: "flags"}},
+	PidfdOpen:                  {{Type: "pid_t", Name: "pid"}, {Type: "unsigned int", Name: "flags"}},
+	Clone3:                     {{Type: "struct clone_args*", Name: "cl_args"}, {Type: "size_t", Name: "size"}},
+	CloseRange:                 {{Type: "unsigned int", Name: "first"}, {Type: "unsigned int", Name: "last"}},
+	Openat2:                    {{Type: "int", Name: "dirfd"}, {Type: "const char*", Name: "pathname"}, {Type: "int", Name: "flags"}, {Type: "mode_t", Name: "mode"}},
```

Comment:
This doesn't fit the man page:
https://man7.org/linux/man-pages/man2/openat2.2.html (which have "how" and "size" arguments)